changelogs/fragments/win_firewall_action.yml

```diff
@@ -0,0 +1,3 @@
+minor_changes:
+- win_firewall - Support inbound, outbound connection in windows firewall.
```

Comment:
Just keep the 1 entry here, the original comment was around making it a bit more descriptive as to what was actually added. Right now seeing `inbound, outbound connection` is a bit vague in what it does. The entry should state something like

> win_firewall - Support defining the default inbound and outbound action of traffic in Windows firewall